shuffle/cli/src/test.rs

```diff
@@ -66,20 +66,38 @@ async fn create_account(
     root_key_path: &Path,
     client: &DevApiClient,
     factory: &TransactionFactory,
+    network: &Network,
 ) -> Result<(Ed25519PrivateKey, LocalAccount)> {
-    let mut treasury_account = account::get_treasury_account(client, root_key_path).await?;
     let account_key = generate_key::generate_key();
+    let new_account = generate_new_account(&account_key, client).await?;
+    match network.get_optional_faucet_url() {
```

Comment:
nit: The return type of `Option<Url>` already indicates that it's optional. Not sure if it's also needed in the method name.